.travis.yml

```diff
@@ -0,0 +1,52 @@
+if: (branch = develop) OR (branch = master) OR (type = pull_request) OR (tag IS present)
+language: node_js
+node_js:
+  - "stable"
+cache:
```

Comment:
nit: why not `cache: node`?

Reply:
I think it's `cache: yarn` or `cache: npm`, I've checked now the docs
I think does exactly the same. No reason, I just copy that from a blog

I'll leave: `cache: yarn`